{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/templates/base.html

```diff
@@ -88,6 +88,13 @@
     ================================================== -->
     <!-- Placed at the end of the document so the pages load faster -->
     {% block javascript %}
+    <!-- BrowserSync debug-only support -->
+    {% if debug %}
+        <script id="__bs_script__">//<![CDATA[
+        document.write('<script async src=\'http://HOST:3000/browser-sync/browser-sync-client.js?v=2.18.8\'><\/script>'.replace('HOST', location.hostname))
```

Comment:
This is 'magic' snippet provided by browserSync (which can be found in its UI, ready for inclusion as is), otherwise it seems to fail to do live updates. No, that should be made available whenever browserSync functionality is required, hence at debugging stage only (in fact, while developing locally).